pkg/querier/partialdata/partia_data.go

```diff
@@ -0,0 +1,19 @@
+package partialdata
+
+import (
+	"errors"
+)
+
+type IsCfgEnabledFunc func(userID string) bool
+
+const ErrorMsg string = "Query result may contain partial data."
```

Comment:
Is it possible to just do?
var PartialDataError = errors.New("query result may contain partial data")

Reply:
Yup, done!